moduleframework/common.py

```diff
@@ -719,3 +718,24 @@ def get_module_type_base():
     if parent not in get_backend_list():
         raise ModuleFrameworkException("As parent is allowed just base type: %s" % get_backend_list)
     return parent
+
+
+def get_docker_file(dir_name="../"):
+    """
+    Function returns full path to dockerfile.
+    :param dir_name: dir_name, where should be Dockerfile located
+    :return: full_path to Dockerfile
+    """
+    fromenv = os.environ.get("DOCKERFILE")
+    if fromenv:
+        dockerfile = fromenv
+        dir_name = os.getcwd()
+    else:
+        dir_name = os.path.abspath(dir_name)
+        dockerfile = DOCKERFILE
+    dockerfile = os.path.join(dir_name, dockerfile)
+
+    if not os.path.exists(dockerfile):
+        dockerfile = None
+        print_debug("Dockerfile should exists in the %s directory." % dir_name)
```

Comment:
this should print absolute path to the directory, relative path will be confusing to users